### DISCUSSION TITLE
closed: received type 3 chunk without previous chunk

### DISCUSSION THREAD

**Discussion by n-endashi** [bluenviron/mediamtx#2836] [Questions]
### Question

I am trying latest mediamtx on Debian 11 and I always get "closed: received type 3 chunk without previous chunk." I am sending to rtmp. I have tried both OBS and Xsplit, same errors. I have rtsp-simple-server 0.19.0 and it is working fine. I compared the config files side by side and everything looks the same.

**Comment by aler9** [comment]
Please open a bug and provide a network dump and server log with option "logLevel" set to debug. The informations you provided are not enough to identify the issue.
